addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -241,6 +247,11 @@ export class MediaDialog extends Component {
                     return true;
                 }));
             }
+            for (const candidateFunc of TABS[this.state.activeTab].Component.mediaExtraClasses) {
+                if (typeof(candidateFunc) === 'function') {
+                    element.classList.add(candidateFunc(this.props));
```

Comment:
Maybe just send the node instead of the whole props object?

Reply:
I was thinking about the case if in the future another functions used in "mediaExtraClasses" need to use a parameter other than a node. Should I only send the node and simplify the function, even though it might require changes in the future ?

Reply:
I guess it's ok :wink: